packages/vm/src/state/baseStateManager.ts

```diff
@@ -497,12 +497,13 @@ export abstract class BaseStateManager {
     const accessList: AccessList = []
     folded.forEach((slots, addressStr) => {
       const address = Address.fromString(`0x${addressStr}`)
-      const check1 = getActivePrecompiles(this._common).find((a) => a.equals(address))
+      // TODO FIXME
+      //const check1 = getActivePrecompiles(this._common).find((a) => a.equals(address))
       const check2 = addressesRemoved.find((a) => a.equals(address))
       const check3 =
         addressesOnlyStorage.find((a) => a.equals(address)) !== undefined && slots.size === 0
 
-      if (!check1 && !check2 && !check3) {
+      if (/*!check1 && */ !check2 && !check3) {
```

Comment:
todo still here

Reply:
I figure the issue here is, you don't have the customPrecompiles that are passed and stored in the vm in the stateManager here. 

you could add it as an optional additional constructor opt and property on the baseStateManager, so they are available if defined

Reply:
have addressed in https://github.com/ethereumjs/ethereumjs-monorepo/pull/1770/commits/b4342f83c06de41e4ef7ef328659c8de283d414c